netshare/generate/denormalize_fields.py

```diff
@@ -32,14 +36,24 @@ def _denormalize_by_fields_list(
     """
     denormalized_data = []
     dim = 0
+
+    canonical_data_dir = get_canonical_data_dir()
+    df, _ = load_dataframe_chunks(canonical_data_dir)
+
     for field in fields_list:
         if is_session_key:
             sub_data = normalized_data[:, dim : dim + field.getOutputDim()]
         else:
             sub_data = normalized_data[:, :, dim : dim + field.getOutputDim()]
+
         sub_data = field.denormalize(sub_data)
-        if isinstance(field, ContinuousField):
-            sub_data = sub_data[:, 0] if is_session_key else sub_data[:, :, 0]
+
+        # For session key, if shape looks like (n, ), change it to (n, 1) for consistency
+        if is_session_key == True and len(sub_data.shape) == 1:
```

Comment:
```suggestion
        if is_session_key and len(sub_data.shape) == 1:
```
Search for "greeting" in https://peps.python.org/pep-0008/

Reply:
(absolutely nit-picking, you can ignore if you wish)

Reply:
It's definitely more elegent. Thanks!!